### PR TITLE
Remove PersistedQueryProvider

### PR DESCRIPTION
**Pull request by bruce-glazier**
## Changes

  - No longer persist certain data to storage

### CODE REVIEW

File: src/common/RootProviders.tsx [hunk 1]
```diff
@@ -24,7 +25,7 @@ export function RootProviders({
   const { apiBaseURL, theme } = useDeveloperConfig();
 
   return (
-    <PersistedQueryProvider>
+    <QueryClientProvider client={new QueryClient()}>
```
Comment (swain) [inline src/common/RootProviders.tsx:28]: We'll want to declare this query client in the top-level global scope -- it needs to be stable for the life of the app, and as-is will re-create on each render.

File: src/common/RootProviders.tsx [hunk 2]
```diff
@@ -12,7 +12,8 @@ import { ActionSheetProvider } from '@expo/react-native-action-sheet';
 import { SafeAreaProvider } from 'react-native-safe-area-context';
 import { ThemedNavigationContainer } from './ThemedNavigationContainer';
 import { LoggedInProviders } from './LoggedInProviders';
-import { PersistedQueryProvider } from '../hooks/PersistedQueryProvider';
+import { QueryClientProvider } from '@tanstack/react-query';
+import { QueryClient } from '@tanstack/query-core';
```
Comment (swain) [inline src/common/RootProviders.tsx:16]: The auto-complete imports have been behaving oddly for me, so I'm not surprised this happened. But, we should import `QueryClient` from `@tanstack/react-query`, not `query-core`.

File: src/common/RootProviders.tsx (hunk 1 above)
Comment (bruce-glazier) [reply src/common/RootProviders.tsx:28]: Gah, totally - I had this sneaking feeling that I was missing something.

**Review by swain — APPROVED** [review]
Nice. Pretty easy refactor!

File: src/hooks/PersistedQueryProvider.tsx [hunk 3]
```diff
@@ -1,62 +0,0 @@
-import { createAsyncStoragePersister } from '@tanstack/query-async-storage-persister';
-import AsyncStorage from '@react-native-async-storage/async-storage';
-import { PersistQueryClientProvider } from '@tanstack/react-query-persist-client';
```
Comment (swain) [inline src/hooks/PersistedQueryProvider.tsx:3]: @bruce-glazier In this PR or a follow-up, I think we can + should remove these two react-query persistence libraries. I think they'll be unused now.

Comment (bruce-glazier) [reply src/hooks/PersistedQueryProvider.tsx:3]: I'll grab it now. Sorry for the slop, extra sleepy today.

**Review by swain — APPROVED** [review]

**Pull request merged**